### ISSUE TITLE
Update Keras Conv2D layer docs to specify padding value

### ISSUE THREAD

**Issue by marcospgp** [tensorflow/tensorflow#43886]
## URL(s) with the issue:
https://keras.io/api/layers/convolution_layers/convolution2d/

## Description of issue (what needs changing):
Specify what value is added as padding when padding argument is set to `same`. Is it zero values?

**Comment by PrattJena** [comment]
Yes in `padding = "same"` we add a n-pixel borders of zero to tell that don't reduce the dimension and have same dimension as input.

**Comment by DARK-art108** [comment]
Can I work on this issue?

**Comment by rmothukuru** [comment]
@marcospgp,
The reason information about `Padding` with Zeroes is not mentioned either in [Keras Conv2D](https://keras.io/api/layers/convolution_layers/convolution2d/) or in [TF Keras Conv2D](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv2D) is because it is an **`Internal Operation on the Input`** of that `Layer` and the `Padded Zeroes` will not be visible to us both in `Input` or `Output Tensors`.

However, information about `Padding` is added via [this PR](https://github.com/tensorflow/tensorflow/pull/40512).

**Comment by rmothukuru** [comment]
@marcospgp,
Can you please respond to the above comment. Thanks!

**Comment by marcospgp** [comment]
@rmothukuru I don't see why it being an internal operation means the user doesn't need to know what's going on, especially since it alters the final result.

If that PR adds the information to the docs then great!

Cheers and thanks for your hard work and care!

**Comment by rmothukuru** [comment]
> @marcospgp,
> The reason information about `Padding` with Zeroes is not mentioned either in [Keras Conv2D](https://keras.io/api/layers/convolution_layers/convolution2d/) or in [TF Keras Conv2D](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Conv2D) is because it is an **`Internal Operation on the Input`** of that `Layer` and the `Padded Zeroes` will not be visible to us both in `Input` or `Output Tensors`.
> 
> However, information about `Padding` is added via [this PR](https://github.com/tensorflow/tensorflow/pull/40512).

@marcospgp,
I take back my comment that it is an internal operation. Yes, it impacts the result. Will update the documentation stating that **`padding = same` pads the Input Image Tensor with Zeros**.
